include/mlir/SPIRV/SPIRVTypes.h

```diff
@@ -52,9 +53,27 @@ enum Kind {
 };
 }
 
+// Provides a common base class between VectorType, SPIR-V ArrayType, SPIR-V
+// StructType.
+class CompositeType : public Type {
+public:
+  using ImplType = DefaultTypeStorage;
```

Comment:
You don't need to specify this, ImplType already defaults to DefaultTypeStorage.

Reply:
@River707 Oh, my fault, thanks! Trying to get more familiar with MLIR.